doc/nrf/ug_bt_mesh_model_config_app.rst

```diff
@@ -0,0 +1,68 @@
+.. _ug_bt_mesh_model_config_app:
+
+Configuring mesh models using the nRF Mesh mobile app
+#####################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth mesh :ref:`samples <samples>` use the nRF Mesh mobile app to perform provisioning and configuration.
+
+Binding a model to an application key
+*************************************
+
+Complete the following steps in the nRF Mesh app to bind a model to an application key:
+
+1. On the Network screen, tap the mesh device node.
+   Basic information about the mesh node and its configuration is displayed.
+#. In the mesh node view, expand an element.
+   It contains the list of models instantiated in the selected element of the node.
+#. Tap the model entry to see the model's configuration.
+#. Bind the model to application key to make it open for communication:
+
+   a. Tap :guilabel:`BIND KEY` at the top of the screen.
+   #. Select the key from the list.
+
+#. Tap :guilabel:`APPLY`, in the right bottom corner of the screen, to confirm the configuration.
+
+Configuring model publication parameters
+****************************************
+
+To configure the model publication parameters, which define how the model will send its messages, complete the following steps in the nRF Mesh app:
+
+1. Tap :guilabel:`SET PUBLICATION` under the :guilabel:`Publish` section.
+#. Tap :guilabel:`Publish Address` under the :guilabel:`Address` section to set the publish address for the node.
+#. Select :guilabel:`Groups` from the drop-down menu.
+#. Select an existing group or create a new one.
+#. Tap :guilabel:`OK`.
+#. If necessary, set the publication interval and publication retransmission configuration.
+   Otherwise, leave the publication parameters at their default values.
+
+   a. Scroll down to the :guilabel:`Publish Period` view and set the publication interval by using the :guilabel:`Interval` slider.
+      This will make the node publish its presence status periodically at the defined interval.
+   b. Change the publication retransmission configuration under the :guilabel:`Publish Retransmission` section. Set the value for the Retransmit Count.
+
+#. Tap :guilabel:`APPLY`, in the right bottom corner of the screen, to confirm the configuration.
+
+Subscribing to a group
+**********************
+
+To configure subscription parameters, complete the following steps in the nRF Mesh mobile app:
+
+1. Tap :guilabel:`SUBSCRIBE` under the :guilabel:`Subscriptions` section.
+#. Select :guilabel:`Groups`.
+#. Select an existing group or create a new one.
+#. Tap :guilabel:`OK`.
+#. Double-tap the back arrow button at the top left corner of the app to get back to the main application screen.
+
+Creating a group
+****************
+
+Complete the following steps in the nRF Mesh mobile app to create a new group:
```

Comment:
Suggestion:
> To send to more than one node, we must define groups for the models to publish and subscribe to.

Also, perhaps this should go above "Subscribing to a group"?

Reply:
Yup, I agree actually. I was a bit back and forth on this one though, but I will move it up.